x-pack/elastic-agent/_meta/config/common.p2.yml.tmpl

```diff
@@ -10,10 +10,14 @@ outputs:
 
 inputs:
   - type: system/metrics
+
+    # The only two requirement are that it has only characters allowed in an Elasticsearch index name
+    # and does NOT contain a `-`.
     dataset.namespace: default
     use_output: default
     streams:
       - metricset: cpu
+        # Same requirements as for the namespace apply.
```

Comment:
I think it would be better to be explicit here as well, what ever is picked above, should just repeat verbatim here.